Week-1/assignment2/style.css

```diff
@@ -0,0 +1,158 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  /* height: 100%;
+  width: 100%; */
+  display: flex;
+  flex-direction: column;
+}
+
+#footer {
+  flex: 1;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #43689b;
+}
+
+#content-container {
+  display: flex;
+  flex-direction: column;
+}
+
+#sideNav {
+  width: 35px;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.side-nav li i {
+  color: #000000;
+  margin: 32px 0px 25px 10px;
+  font-weight: bold;
+}
+
+.description {
+  margin-left: 35px;
+  padding: 20px;
+  display: flex;
+  flex-direction: row;
+  background-color: #454545;
+  color: aliceblue;
+}
+
+.description img {
+  width: 200px;
+  height: 200px;

```

Comment:
instead of setting up both width and height, try setting only one of them and see how the other property changes. 
Setting both of them will update the aspect ratio of the image and it might appear pixelated. This is generally not the experience the user wants unless specifically required.